charts/traffic-templates/templates/traffic.yaml

```
@@ -1,20 +1,56 @@
 {{- $versions := include "resolve.modelVersions" . | mustFromJson }}
 
+{{- /* validate values for templateName */}}
+{{- if not (has .Values.templateName (list "initialize-rollout" "modify-weights")) }}
+  {{- printf "Unknown templateName: '%s'" .Values.templateName | fail }}
+{{- end}}
```

```suggestion
{{- end }}
```

I see a few other instances.

---

fixed